doc/upgrading/upgrade-v2.4-to-v3.0.md

```diff
@@ -37,3 +37,22 @@ _Applies To: Projects with a custom `SoftDeleteableAdapter` implementation_
 
 The [`SoftDeleteableAdapter`](/src/SoftDeleteable/Mapping/Event/SoftDeleteableAdapter.php) interface has a new method
 for generating the DateTime value.
+
+## Translatable
+
+_Applies To: Projects migrated from Symfony v4 to v5_
```

Comment:
Where is the base documentation for this integration? What if there is no involved migration? Is it not required for users of Symfony v5?

Reply:
Base doc: https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/doc/symfony4.md

Actually it is required for all of the Symfony v5 users, tho there is no base documentation for Syfmony 5.